doc/test-suite-diff.py

```diff
@@ -128,12 +128,19 @@ def end_webpage(html, body, htmlOutFile):
             name, ext = os.path.splitext(item2)
             pngFile1 = os.path.join(path_in1, item1, name + '.png')
             pngFile2 = os.path.join(path_in2, item1, name + '.png')
+            jsonFile1 = os.path.join(path_in1, item1, name + '.json')
+            jsonFile2 = os.path.join(path_in2, item1, name + '.json')
             pngFileOut = os.path.join(path_out, item1, name + '.png')
             pngFile1Out = os.path.join(path_out, item1, name + '.after.png')
             pngFile2Out = os.path.join(path_out, item1, name + '.before.png')
             print(f'Comparing {name}')
 
-            diffValue = diffimg.diff(pngFile1, pngFile2, delete_diff_file=True)
+            timeMap1 = json.load(open(jsonFile1, 'r'))
+            timeMap2 = json.load(open(jsonFile2, 'r'))
+            if jsondiff(timeMap1, timeMap2):
```

Comment:
Since you're not actually doing anything with the diff, you're only checking to see if they're different, you could probably get away with using the `similarity` function instead. This seems to do a bit less processing, and just returns a number indicating the similarity which is all you need. This also avoids implicitly checking the truthyness of a Dict result, which I generally avoid because it has some unexpected behaviours in some cases.

```
from jsondiff import similarity as jsonsimilar

... 

if jsonsimilar(t1, t2) < 1.0:
    print("...")
```

Dictionary truthiness:

```
>>> {} is False
False
>>> {} is True
False
>>> {} is None
False
>>> if {}: print("hello world")
>>> if not {}: print("hello world")
hello world
>>>